qiskit/transpiler/passes/basis/basis_translator.py

```diff
@@ -207,7 +207,7 @@ def run(self, dag):
                     "target basis is not universal or there are additional equivalence rules "
                     "needed in the EquivalenceLibrary being used. For more details on this "
                     "error see: "
-                    "https://docs.quantum-computing.ibm.com/api/qiskit/qiskit.transpiler.passes."
+                    "https://docs.quantum.ibm.com/api/qiskit/qiskit.transpiler.passes."
```

Comment:
```suggestion
                    "https://docs.quantum.ibm.com/api/qiskit/transpiler_passes."
```

the link was bust before as well